matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/data/LoginFlowResponse.kt

```diff
@@ -34,5 +35,13 @@ internal data class LoginFlow(
          * The login type. This is supplied as the type when logging in.
          */
         @Json(name = "type")
-        val type: String?
+        val type: String?,
+
+        /**
+         * Augments m.login.sso flow discovery definition to include metadata on the supported IDPs
+         * the client can show a button for each of the supported providers
+         * See MSC #2858
+         */
+        @Json(name = "identity_providers")
+        val identityProvider: List<IdentityProvider>?
```

Comment:
Please rename to `identityProviders` (or any other name if the type is renamed, but ending with an `s`)

Reply:
(Done?)